server/routes/pt.js

```diff
@@ -114,4 +116,184 @@ router.post('/search', verifyToken, async (req, res) => {
   }
 });
 
+router.post('/addptrequest', verifyToken, async (req, res) => {
+  try {
+    const pt_id = req.body.pt_id; // ID of PT that was requested
+    const client_id = req.user.userId; // ID of client that made request
+ 
+    Pt.findByIdAndUpdate(pt_id,
+      {$push: {requests: client_id}},
+      {safe: true, upsert: true},
```

Comment:
```suggestion
      {safe: true, upsert: false},
```
don't want to make them a pt if they aren't one already.